fvm/transaction.go

```diff
@@ -77,14 +80,66 @@ func (i *TransactionInvocator) Process(
 
 	location := common.TransactionLocation(proc.ID[:])
 
+	var predeclaredValues []runtime.ValueDeclaration
+
+	if ctx.AccountFreezeAvailable {
+
+		setAccountFrozen := runtime.ValueDeclaration{
+			Name: "setAccountFrozen",
+			Type: &sema.FunctionType{
+				Parameters: []*sema.Parameter{
+					{
+						Label:          sema.ArgumentLabelNotRequired,
+						Identifier:     "account",
+						TypeAnnotation: sema.NewTypeAnnotation(&sema.AddressType{}),
+					},
+					{
+						Label:          sema.ArgumentLabelNotRequired,
+						Identifier:     "frozen",
+						TypeAnnotation: sema.NewTypeAnnotation(&sema.BoolType{}),
+					},
+				},
+				ReturnTypeAnnotation: &sema.TypeAnnotation{
+					Type: &sema.IntType{},
```

Comment:
This should probably return `Void`, i.e. 
```suggestion
					Type: sema.VoidType,
```